src/spoke/Spoke.sol

```diff
@@ -31,6 +31,8 @@ import {ISpokeGatewayHandler} from "../common/interfaces/IGatewayHandlers.sol";
 import {VaultUpdateKind, MessageLib} from "../common/libraries/MessageLib.sol";
 import {IPoolEscrowFactory} from "../common/factories/interfaces/IPoolEscrowFactory.sol";
 
+IRequestManager constant REQUEST_MANAGER_V3_0 = IRequestManager(0x58d57896EBbF000c293327ADf33689D0a7Fd3d9A);
```

Comment:
CR: The live AsyncRequestManager from v3.0.1 is `0xF06f89a1b6C601235729A689595571B7455dD433` ([ref](https://docs.centrifuge.io/developer/protocol/deployments/#mainnet))